cli/args/flags.rs

```diff
@@ -751,6 +758,10 @@ and report results to standard output:
 
   deno bench src/fetch_bench.ts src/signal_bench.ts
 
+Output benchmark result in JSON format:
```

Comment:
This isn't really needed I think, the `--help` will print out the individual args either way so this will just be redundant information.

Reply:
Removed